test/es-module/test-esm-import-wasm.mjs

```diff
@@ -0,0 +1,15 @@
+// Flags: --experimental-modules --harmony-dynamic-import
+
+import { crashOnUnhandledRejection } from '../common';
+import assert from 'assert';
+
+crashOnUnhandledRejection();
+
+import { add } from '../fixtures/es-modules/add.wasm';
+
+assert.strictEqual(add(2, 3), 5);
+
+import('../fixtures/es-modules/add.wasm').then((ns) => {
```

Comment:
I'm guessing that this will get the module from the cache and not instantiate it. This is good, as it checks wasm and cache together, but unfortunately it doesn't check dynamic importing a wasm module.

Perhaps add another `import('../fixtures/es-modules/add.wasm?cache-avoider').then(...)`?

Reply:
i'm not checking dynamic instantiation (we know that works), i'm checking the namespace and default exports